orchestra/static/orchestra/todos/todo-checklist.html

```diff
@@ -28,11 +29,14 @@
       </span>
       <span ng-if="templates.length > 0">
         &nbsp;&nbsp;
-        <a href="#" ng-hide="!showSkipped" ng-click="unskipTodo(todo)">[add]</a>
-        <a href="#" ng-hide="showSkipped" ng-click="skipTodo(todo)">[skip]</a>
+        <a href="#" ng-hide="!showSkipped" ng-click="unskipTodo(todo)">[relevant]</a>
+        <a href="#" ng-hide="showSkipped" ng-click="skipTodo(todo)">[not relevant]</a>
         <a href="#" ng-hide="todo.parent_todo" ng-click="removeTodo(todo)">[remove]</a>
       </span>
     </label>
+    <form class="form-inline" ng-if="recommendations[todo.description] && recommendations[todo.description].approval_reason" >
```

Comment:
can't we use todo.id here?

Reply:
We cannot because the primary use case is to show the recommendation on the next project where todo ids will not match.